react/training-project/src/app/screens/Profile/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import PrimaryLayout from '~components/PrimaryLayout';
+
+import ProfileDumb from './layout';
+
+function Profile({ userdata }) {
```

Comment:
When you have an index.js and a layout.js files, call the component in the first one `MyComponentContainer` and the one in the second file `MyComponent.`. In this case, you should make this change:
`Profile` -> `ProfileContainer`
`ProfileDumb` -> `Profile`